core/vault-worker.go

```diff
@@ -137,7 +137,7 @@ func vaultInit(config *tomlConfig, httpClient *http.Client, debug bool) (sCode i
 	}
 
 	// Save the JSON structure to a file system JSON file
-	err = ioutil.WriteFile(config.SecretService.TokenFolderPath+"/"+config.SecretService.VaultInitParm, initRequestResponseBody, 0600)
+	err = ioutil.WriteFile(config.SecretService.TokenFolderPath+"/"+config.SecretService.VaultInitParm, initRequestResponseBody, 0644)
```

Comment:
Just FYI that `0644` will create a generally read/writable file on Windows.

https://golang.org/pkg/os/#Chmod

Reply:
This is due to the reason that mongoDB script needs to read the file in the shared volume folder. eventually mongoDB script will be replaced with golang implementation and the file will be encrypted.

Reply:
no long valid as mongoDB script will be out of scope.